Assets/Scripts/Utility/RotateObject.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 9f7f072c827ea4546989fb95f9548627
+guid: f492d2ccc97634d44ba5b241f141deb2
```

Comment:
Why did this GUID change?

Reply:
It was changed _back_ to what it was originally. After moving it in a previous commit, the meta file wasn't moved properly so the file got a new guid. I manually fixed (some) of the resulting linkages when I realized the problem, but then @arthur740212 pointed out that the _right_ way to do this is really to change the guid back to what it was originally. Whew - that was convoluted sorry - but hopefully it's followable.